apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.html

```diff
@@ -0,0 +1,36 @@
+<fd-card cardType="analytical">
+    <fd-card-header
+        >Describes the type of
```

Comment:
I think this line might be a copy/paste error or something

Reply:
Fixed